docs/CONFIG-VARS.md

```diff
@@ -198,7 +198,7 @@ Note: The 128 default is in GB, so with a RAID5, the default is 4 disks, [so the
 | :--- | ---: | ---: | ---: | ---: |
 | create_nfs_public_ip | Add public ip to the NFS server VM | bool | false | |
 | nfs_vm_admin | OS Admin User for the NFS server VM | string | "nfsuser" | |
-| nfs_vm_machine_type | SKU to use for NFS server VM | string | "Standard_B2s_v3" | To check for valid types for your subscription `az vm list-skus --resource-type virtualMachines --subscription $subscription --location $location -o table`|
+| nfs_vm_machine_type | SKU to use for NFS server VM | string | "Standard_D8s_v4" | To check for valid types for your subscription `az vm list-skus --resource-type virtualMachines --subscription $subscription --location $location -o table`|
```

Comment:
NFS VM type should be D-series (https://github.com/sassoftware/viya4-iac-azure/blob/main/variables.tf#L283). B-series is only for Jump VM.

Reply:
Yes, that's is the default for the NFS VM. It was documented incorrectly. The adjustment here has fixed that issue.